docs/scale_run_script.md

```diff
@@ -0,0 +1,71 @@
+# Using the scale run script
+
+The [utils/scale_run.sh](../utils/scale_run.sh) script is used to automate the
+execution of scale testing for the various EC2 instance types.  This script
+runs 5 rounds each of cold start and warm start scenarios.  It creates a
+directory for each run.
+
+Out of the box, it will run the following:
+
+* 1 run of 100 instances on a c5.large instance for the trial license
+  (10 hosts) scenario.  Tuning is set to true, but is not forced, so it will
+  not be applied to this instance due to insufficient ram.
+* 5 cold start runs (provisioning on the first) with tune set to true on the
+  following:
+    * c5.xlarge: Starting with 2000 instances
```

Comment:
I would avoid putting instance counts here so they don't get out of sync with the code... Maybe just add a line that the instance counts are defined at the top of the file or something.

Reply:
Doc updated to avoid using instance counts.